python/GafferCyclesUI/CyclesShaderUI.py

```diff
@@ -84,7 +84,12 @@ def __translateParamMetadata( nodeTypeName, socketName, value ) :
 		__metadata[paramPath]["plugValueWidget:type"] = "GafferUI.PresetsPlugValueWidget"
 
 	if( socketName == "filename" ) :
-		__metadata[paramPath]["plugValueWidget:type"] = "GafferUI.PathPlugValueWidget"
+		__metadata[paramPath]["plugValueWidget:type"] = "GafferUI.FileSystemPathPlugValueWidget"
+		__metadata[paramPath]["path:leaf"] = True
+		__metadata[paramPath]["path:valid"] = True
+		__metadata[paramPath]["path:bookmarks"] = "texture"
+		__metadata[paramPath]["fileSystemPath:extensions"] = "exr tif tiff hdr jpg jpeg png"
```

Comment:
When I did the equivalent for Arnold, I only included the `tx` extension, to encourage folks to use properly mip-mapped textures. Perhaps it's worth doing the same here? Or perhaps at the very least we could include `tx` in this list? I _think_ Cycles uses OIIO for image loading, so if we wanted a complete list we could get it from `GafferImageImageReader.supportedExtensions()`.